### PR TITLE
[WIP][ENH] mne_prepare_bem_model command

### PR DESCRIPTION
**Pull request by vferat**
## Reference issue

Part of discussion in #6707
## What does this implement/fix?

This PR adds a command line function **mne_prepare_bem_model** that wraps [**mne.make_bem_solution** ](https://mne.tools/stable/generated/mne.make_bem_solution.html) and save its ouput using [**mne.write_bem_solution**](https://mne.tools/stable/generated/mne.write_bem_solution.html#mne.write_bem_solution).

It tries to replicate the [MNE-C API](https://mne.tools/0.17/manual/c_reference.html#mne-prepare-bem-model).

### CODE REVIEW

File: mne/commands/mne_prepare_bem_model.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    if bem_fname is None:
+        parser.print_help()
+        sys.exit(1)
+
+    if bem_sol_fname is None:
+        base, _ = os.path.splitext(bem_fname)
+        bem_sol_fname = base + '-sol.fif'
+
+    bem_model = mne.read_bem_surfaces(bem_fname, patch_stats=False,
```
Comment (vferat) [inline mne/commands/mne_prepare_bem_model.py]: No idea if patch_stats should be set to True/False or a sohuld be  command line argument.

File: mne/commands/mne_prepare_bem_model.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                      help='Specify the name of the file containing the '
+                           'triangulations of the BEM surfaces and the '
+                           'conductivities of the compartments. The standard '
+                           'ending for this file is -bem.fif.',
+                      metavar="FILE")
+    parser.add_option('--sol', dest='bem_sol_fname',
+                      help='Specify the name of the file containing the'
+                           'triangulation and conductivity information '
+                           'together with the BEM geometry matrix computed by '
+                           'mne_prepare_bem_model . The standard ending for '
```
[the remark this answers is not in the document]
Comment (agramfort) [reply mne/commands/mne_prepare_bem_model.py]: this description is actually wrong. You are computing here the "geometry matrix of the BEM".

I would say

```
help='The name of the file containing BEM solution (geometry matrix). It uses the linear collocation approach. The file should end with -bem-sof.fif.'
```

File: mne/commands/mne_prepare_bem_model.py (hunk 1 above)
Comment (larsoner) [reply mne/commands/mne_prepare_bem_model.py]: Let's just go with the default for now and we can add the argument later if we need it

**Comment by larsoner** [comment]
Thanks @vferat

**Pull request merged**